server/channels/web/handlers.go

```diff
@@ -409,7 +409,7 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		if IsAPICall(c.App, r) || IsWebhookCall(c.App, r) || IsOAuthAPICall(c.App, r) || r.Header.Get("X-Mobile-App") != "" {
 			w.WriteHeader(c.Err.StatusCode)
-			w.Write([]byte(c.Err.ToJSON()))
+			w.Write([]byte(c.Err.ToJSON(*c.App.Config().ServiceSettings.EnableDeveloper)))
```

Comment:
I don't understand; we are already doing it in L397. What is the difference?

Reply:
It is being re-added in `ToJSON` because of https://github.com/mattermost/mattermost/pull/20846

Reply:
Ah yes my bad! Ok, so then I think we need to replace all occurences of `c.Err.DetailedError = ""` with `c.Err.WipeDetail()` where the method wipes off both `DetailedError` and `wrappedError`.

Reply:
@agnivade Ahhh, that's a _much_ nicer solution, thanks.  I also made the changes for websocket errors, but I'm not totally sure if we need to. What do you think?